Readme.md

```diff
@@ -13,4 +15,6 @@ Some common workflows and actions for Github Actions shared between Lombiq proje
 
 Bug reports, feature requests, comments, questions, code contributions and love letters are warmly welcome. You can send them to us via GitHub issues and pull requests. Please adhere to our [open-source guidelines](https://lombiq.com/open-source-guidelines) while doing so.
 
+We recommend using [Visual Studio Code](https://code.visualstudio.com/) with the (unofficial) [GitHub Actions](https://marketplace.visualstudio.com/items?itemName=me-dutour-mathieu.vscode-github-actions) extension (until [the official one](https://github.com/github/roadmap/issues/564) is released).
```

Comment:
This is better suited for https://lombiq.atlassian.net/wiki/spaces/DEV/pages/2545614855/Writing+GitHub+Actions+workflows+and+actions (we don't just write GHA code in this repo). Also include any notable rules for things you made uniform.

Reply:
Update: There are some side-effects to what I wanted to set up:

- When you set `Prose Wrap` to `always` and `Print Width` to `120`, it will correctly wrap lines the way we want, but auto-formatting in the unofficial GHA extension is not entirely context-aware and it (figuratively and literally) breaks code lines too, not just descriptions, so that's a no-go.
- `Prose Wrap` set to `preserve` works if you go over the line width but doesn't rewrap the text if a line still has space for text from the next line.
- But the Rewrap extensions still handles this correctly, because it's just text!

So, my recommendation:

1. Use the unofficial GHA extension, because it's great for schema validation and IntelliSense, but leave the wrapping settings on default, because we don't actually use them.
2. Use Rewrap (that's already in our VS Code guidelines) for wrapping descriptions.
3. The previous also makes it necessary to start one-line descriptions on a new line (and put ` <` after `description:`), because Rewrap isn't context-aware either, so if the description grows over the length limit (e.g., you start with `description: Hello Lombiq!` and expand it), then wrapping will kick in and also move `description:` to the previous line.
This is not a problem when the description is one line but starts with `<` and the text is on the next line or when the text is already multi-line (regardless of having `<` or not).
4. To generalize the previous point, I'd rather just say that description should always have the `<` and start the text on the next line.

Reply:
Adding a character to the code for step 3 that otherwise isn't needed just to make an IDE plugin work correctly is not something I'd do. Since wrapping is a trivial thing anyway (and works with Rewrap as you also point out), and it's not needed for single-line descriptions (if you move to multi-lines then wrapping works anyway) I don't see why we'd set this up.

Reply:
My though process was more about consistency, i.e., to enforce a styling that always looks the same (not just for the sake of extension not tripping up but regarding YAML-syntax in general too). The problem is more about not being able to enforce it and GHA doesn't care how we fold descriptions (they won't be rendered anywhere either).

Reply:
We can perhaps have proper enforcement with https://github.com/Lombiq/NodeJs-Extensions/issues/11.

Reply:
Yep, Oliver commented this on #169, that would be a great addition!